content/en/docs/how-tos/notification.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Get notifications about a job"
+date: 2020-01-06T10:08:01-04:00
+draft: false
+---
+
+The `reporter_config` stanza in a job can be used to send notifications. [For example](https://github.com/openshift/release/blob/964a2cb316ab046a8c3d856d9c99cb40bfbbef2c/ci-operator/jobs/openshift/odo/openshift-odo-master-periodics.yaml#L2-L24), the following configuration will trigger a slack notification to channel `#odo-notifications` in [coreos.slack.com](https://coreos.slack.com/) when the job is at state `success`, `failure`, or `error`. The `report_template` defines the template of the slack message.
```

Comment:
Few items:

1. Don't link to a real config, the doc should have an inline example that's clear enough and works - if we have it, there's no need linking to somethign that's potentially stale.
2. Please invest a bit more effort into the text - I'd like to reiterate Steve's feedback here: https://github.com/openshift/ci-docs/pull/47#pullrequestreview-561216115. The doc should go from generic (It is possible to set up Slack notifications by editing Prowjob yaml to specific (how exactly to edit what yaml). It think there should be a paragraph about limitations (only coreos slack, useful mostly for periodics, supported only on generated periodics (and not generated presubmits and postsubmits)

Reply:
Yes. I noticed the problem(s) although I do not have a solution yet.
My plan is to write more docs to improve doc writing and know more technical details about our platform.